plugins/gimp/file-jxl-save.h

```diff
@@ -6,15 +6,61 @@
 #ifndef PLUGINS_GIMP_FILE_JXL_SAVE_H_
 #define PLUGINS_GIMP_FILE_JXL_SAVE_H_
 
-#include <libgimp/gimp.h>
-
-#include "lib/jxl/base/status.h"
+#include "jxl/encode.h"
+#include "jxl/encode_cxx.h"
+#include "plugins/gimp/common.h"
 
 namespace jxl {
 
-Status SaveJpegXlImage(gint32 image_id, gint32 drawable_id,
-                       gint32 orig_image_id, const gchar* filename);
+bool save_dialog();
+
+bool SaveJpegXlImage(gint32 image_id, gint32 drawable_id, gint32 orig_image_id,
+                     const gchar* filename);
+
+class JpegXlSaveOpts {
+ public:
+  float distance;
+  float quality;
```

Comment:
Which one ends up getting used? `distance` or `quality`? I see that in the dialog you have both...

Reply:
Oh, nvm me, apparently they are tied together :)

Reply:
My favorite part of the plugin is those sliders.